packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -252,6 +257,9 @@ export default (async function PgIntrospectionPlugin(
         memo[x[attrKey]][x[attrKey2]] = x;
         return memo;
       }, {});
+
+    //TODO seems like a confusing namespace convention to put id lookups under kind lookups
```

Comment:
A "namespace" is postgres' internal name for a schema; it comes from the same introspection query as everything else, just kind === namespace.

Reply:
I was actually referencing the fact that all of the "XbyId" indexes are under the pgIntrospectionResultsByKind object. I thought it might be better to have them under a "pgIntrospectionResultsByID" object. I'll add more on my thoughts below.